crates/storage/libmdbx-rs/src/environment.rs

```diff
@@ -767,6 +773,20 @@ impl EnvironmentBuilder {
         self
     }
 
+    /// Controls interprocess/shared threshold to force flush the data buffers to disk, if
+    /// [`SyncMode::SafeNoSync`] is used.
+    pub fn set_sync_bytes(&mut self, v: usize) -> &mut Self {
+        self.sync_bytes = Some(v as u64);
+        self
+    }
+
+    /// Controls interprocess/shared relative period since the last unsteady commit to force flush
```

Comment:
just for consistency with other methods

```suggestion
    /// Sets the interprocess/shared threshold to force flush the data buffers to disk, if
    /// [`SyncMode::SafeNoSync`] is used.
    pub fn set_sync_bytes(&mut self, v: usize) -> &mut Self {
        self.sync_bytes = Some(v as u64);
        self
    }

    /// Sets the interprocess/shared relative period since the last unsteady commit to force flush
```